tests/test_schema_syntax.py

```diff
@@ -111,7 +111,7 @@ def test_eschema_syntax_semicolon_02(self):
     def test_eschema_syntax_semicolon_03(self):
         """
         abstract type test::OwnedObject {
-            required link owner -> User;
+            required link owner -> User
```

Comment:
Can you add a test with a default expression? Also test_eschema_syntax tests all have semicolons in them, can you randomly drop a few from a few tests?